plugin/trino-openlineage/src/test/java/io/trino/plugin/openlineage/http/TestOpenLineageHttpTransportConfig.java

```diff
@@ -51,7 +52,7 @@ void testExplicitPropertyMappings()
                 .put("openlineage-event-listener.transport.timeout", "30s")
                 .put("openlineage-event-listener.transport.headers", "header1:value1,header2:value2")
                 .put("openlineage-event-listener.transport.url-params", "urlParam1:urlVal1,urlParam2:urlVal2")
-
```

Comment:
How about this cleanup as a prep commit -

Reply:
IMHO creating a commit which removes one empty line is too much